### DISCUSSION TITLE
Cara rename legend title dan label

### DISCUSSION THREAD

**Discussion by reyhanpurnawan** [teamalgoritma/community#433] [Q&A] [answered]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/88662517/199014225-343786fb-f515-40f9-b2b9-6a3785b88248.png">
[Shiny CP.zip](https://github.com/teamalgoritma/community/files/9901238/Shiny.CP.zip)

**Comment by jafarhf1** [comment]
Dear @reyhanpurnawan ,
Untuk merename legend title dan labelnya bisa ditambahkan fungsi `scale_color_manual()` dengan menggunakan parameter `name=` (untuk title) dan `labels=` (untuk label).

Semoga membantu.

**Reply by reyhanpurnawan** [reply]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/88662517/199031904-f717bbe4-8200-42ae-bbf8-9d5ea5c54e19.png">
`scale_color_manual(name = "", labels = c("x", "y"))`
[Shiny CP.zip](https://github.com/teamalgoritma/community/files/9901978/Shiny.CP.zip)
belum bisa mas

**Reply by reyhanpurnawan** [reply]
[Shiny CP.zip](https://github.com/teamalgoritma/community/files/9901986/Shiny.CP.zip)

**Comment by jafarhf1** [comment, marked as answer]
Berdasarkan file yang bapak berikan, alternatif lainnya adalah bapak bisa langsung mendefinisikan label legendnya dari parameter `color` didalam fungsi `geom_line()` nya contohnya sebagai berikut:
```
geom_line(aes(y = local_price, color = "Local Price"))
```
Kemudian untuk mengganti title legendnya menggunakan fungsi 
```
scale_color_discrete(name ="")
```
Semoga membantu.

**Reply by reyhanpurnawan** [reply]
Terima kasih mas
